contracts/random/README.md

```diff
@@ -6,7 +6,7 @@ The reasons for using these contracts are that:
 
 * Enables you to leverage a random number generation system designed by Immutable's cryptographers.
 * Allows you to build your game against an API that won't change.
-* The quality of the random numbers generated will improve as new capabilities are added to the platform. That is, the migration from ```block.hash``` to ```block.prevrandao``` when the BFT fork occurs will be seamless.
+* The quality of the random numbers generated will improve as new capabilities are added to the platform. 
```

Comment:
What further improvements might there be?

Reply:
I will reword this. There are two things: 1. the block hash to prevrandao, and 2. maybe there will be bug fixes or other improvements. These improvements will (hopefully) happen without changing the API. Hence, people who implement based on the API will get fixes for free.

Reply:
I have addressed this comment by updating the wording.